src/app/debug/table/table.component.html

```diff
@@ -111,18 +96,29 @@ <h4 class="progress-report-timout">[One or more reports are in progress for more
           </button>
         }
       }
-      <app-active-filters class="ml-2" [activeFilters]="currentFilters" (click)="toggleFilter()"></app-active-filters>
+      <app-active-filters class="ml-2" [activeFilters]="tableSettings.currentFilters" (click)="toggleFilter()"></app-active-filters>
     </div>
-    <div data-cy-debug="table" class="table-responsive" id="metadataTable">
-      <table data-cy-debug="tableBody" mat-table class="table mb-0" matSort [style.font-size]="getFontSize()" [dataSource]="tableDataSource">
+    @if (hasTimedOut) {
+      <h4 class="progress-report-timout">[One or more reports are in progress for more
```

Comment:
```suggestion
      <h4 class="progress-report-timeout">[One or more reports are in progress for more
```
typo fix for previous comment

Reply:
fixed